styles.css

```diff
@@ -1 +1,76 @@
-/*Maak hier jouw CSS!*/
\ No newline at end of file
+/*Dit is dan voor alles kleiner dan 576px, dus voor mobiel*/
+.grid-container {
+    background-color: #EAF2D7;
+    display: grid;
+    row-gap: 4px;
```

Comment:
In de opdracht stond dat we regelafstanden voor zowel de rijen als de kolommen op 16px moesten hebben voor zowel mobiel als desktop. Deze nog even aanpassen.